### PR TITLE
✨ Add utility functions and snippets to javascript.json

### PR DESCRIPTION
**Pull request by realvishalrana**
image

### CODE REVIEW

File: public/consolidated/all_snippets.json
Comment (Mathys-Gasnier) [inline public/consolidated/all_snippets.json:1]: `all_snippets.json` shouldn't be modified as it is automaticly generated, please remove your changes to it.

**Review by Mathys-Gasnier — CHANGES_REQUESTED** [review]

File: public/consolidated/all_snippets.json [hunk 1]
```diff
@@ -371,6 +371,99 @@
...
+      }
+    ]
+  },
+  {
+    "language": "javascript",
+    "categoryName": "Object Manipulation",
+    "snippets": [
+      {
+        "title": "Filter Object",
+        "description": "Filters out entries in an object where the value is falsy.",
```
Comment (psychlone77) [inline public/consolidated/all_snippets.json]: Small spelling mistake in the description

File: public/consolidated/all_snippets.json [hunk 2]
```diff
@@ -371,6 +371,99 @@
...
+        "code": [
+          "export const filterObject = (object = {}) =>",
```
Comment (psychlone77) [inline public/consolidated/all_snippets.json]: This function is useful, but filters out entries if they are 0 , "", ''. I'm not sure if that's what you intended. Maybe you can directly check for null, undefined values if not 🤔 .

File: public/consolidated/all_snippets.json (hunk 1 above)
Comment (realvishalrana) [reply public/consolidated/all_snippets.json]: @psychlone77 change Filters  to filter

**Review by Mathys-Gasnier — APPROVED** [review]

**Review by psychlone77 — APPROVED** [review]
Looks good. Also ngl these snippets are very useful 😅

File: public/consolidated/all_snippets.json (hunk 1 above)
Comment (psychlone77) [reply public/consolidated/all_snippets.json]: The spelling mistake I thought was for the word 'falsy', which I learned today is a word in the JavaScript world 😅😅

**Pull request merged**